tests/bsd/color.sh

```diff
@@ -0,0 +1,9 @@
+ORI_LSCOLORS="$LSCOLORS"
+ORI_CLICOLOR="$CLICOLOR"
+export CLICOLOR="YES"
+export LSCOLORS="eb"
+invoke_bfs basic -color -maxdepth 1 >"$OUT"
```

Comment:
This should just use `bfs_diff` I think.  It should also probably use the `rainbow` test directory.  See the other `tests/bfs/color*.sh` tests for examples.